CSharp_Course_Project/Task2_PrinterError.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Course_Project
+{
+    class Task2_PrinterError
+    {
+        public static void PrinterError()
+        {
+            string controlString = GetControlString();
+            double errorCounter = GetErrorCounter();
+
+            Console.WriteLine(controlString);
+            Console.WriteLine($"{errorCounter} / {controlString.Length}");
+            double result = errorCounter / controlString.Length;
+            Console.WriteLine($"The error rate = {result}");
+        }
+
+        private static string GetControlString()
+        {
+            var random = new Random();
+            int randomNumber = random.Next(10, 50);
+
+            char[] chars = "abcdefghijklmnopqrstuvwxyz".ToCharArray();
+            char[] printerColors = new char[randomNumber];
+
+            for (int i = 0; i < printerColors.Length; i++)
+            {
+                printerColors[i] = chars[random.Next(chars.Length)];
+            }
+            Array.Sort(printerColors);
+            string controlString = string.Join("", printerColors);
```

Comment:
No need to join the array of chars to string due to the string is already an array of chars.